website/docs/quick-start.md

```diff
@@ -131,4 +131,65 @@ This sample illustrates how to patch containers using vulnerability reports with
     2022/05/16 18:00:17 [notice] 31#31: signal 28 (SIGWINCH) received
     ```
 
-   You can stop the container by opening a new shell instance and running: `docker stop nginx-test`
\ No newline at end of file
+   You can stop the container by opening a new shell instance and running: `docker stop nginx-test`
+
+## Filtering Vulnerabilities
```

Comment:
can we create a new top level bullet (similar to quick start - see https://github.com/project-copacetic/copacetic/blob/main/website/sidebars.js) called `Troubleshooting` or similar and add this content there?

Reply:
Yess, that will be perfect!
Thanks, will do that